src/main/java/com/bandwidth/sqs/action/SetQueueAttributesAction.java

```diff
@@ -7,17 +7,17 @@
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesRequestMarshaller;
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesResultStaxUnmarshaller;
 import com.bandwidth.sqs.action.adapter.SqsAwsSdkAction;
-import com.bandwidth.sqs.queue.SqsQueueAttributes;
+import com.bandwidth.sqs.queue.SqsQueueAttributeChanges;
```

Comment:
The more I looked through the code, the more I believe it should be the former: attributes.
`ChangedSqsQueueAttributes`, `SqsQueueChangedAttributes`, ...